test/parallel/test-child-process-execfile.js

```diff
@@ -2,10 +2,10 @@
 const common = require('../common');
 const assert = require('assert');
 const execFile = require('child_process').execFile;
-const path = require('path');
 const uv = process.binding('uv');
+const { fixturesPath } = require('../common/fixtures');
 
-const fixture = path.join(common.fixturesDir, 'exit.js');
+const fixture = fixturesPath('exit.js');
```

Comment:
I don't like losing the `common.` prefix, I think the `const { fixturesPath }` is a really nice syntax, but when you're going through tests it's really helpful to know which variables are `common`. Assuming the movement towards breaking up `common` continues, we'll have more and more variables that aren't obviously from `common`.

I think the options are:

1. What this PR currently does, import the functions you want to use directly from `fixtures`, so you use with `fixturesPath()`.
2. Don't use the `{ fixturesPath }` require syntax, so you do `const fixtures = require('../common/fixtures)`, and use with `fixtures.fixturesPath()` (or maybe just `fixtures.path()`).
3. Import and re-export each function/object from the common module, so you do `common.fixturesPath()`.
4. Import and re-export `fixtures` from common, so you do `common.fixtures.fixturesPath()` (or `common.fixtures.path()`).

I think I'd prefer `4.`, interested to see what other people think.